### PR TITLE
Split the Capabilities by endpoints

### PR DESCRIPTION
**Pull request by ola-rozenfeld**
Fixes https://github.com/bazelbuild/remote-apis/issues/61.
Will need to fix Bazel to match!

### CODE REVIEW

File: build/bazel/remote/execution/v2/remote_execution.proto [hunk 1]
```diff
@@ -332,7 +332,14 @@ service ContentAddressableStorage {
 // The query may include a particular `instance_name`, in which case the values
 // returned will pertain to that instance.
 service Capabilities {
-  // GetCapabilities returns the server capabilities configuration.
+  // GetCapabilities returns the server capabilities configuration of the
+  // remote endpoint.
+  // Only the capabilities of the services supported by the endpoint will
+  // be returned. For example:
```
Comment (buchgr) [inline build/bazel/remote/execution/v2/remote_execution.proto]: Is 'For example' necessary here? Couldn't we simply specify an exhaustive list?

File: build/bazel/remote/execution/v2/remote_execution.proto [hunk 2]
```diff
@@ -1301,9 +1308,13 @@ message GetCapabilitiesRequest {
 // [Capabilities.GetCapabilities][build.bazel.remote.execution.v2.Capabilities.GetCapabilities].
 message ServerCapabilities {
   // Capabilities of the remote cache system.
+  // This field will be returned by endpoints that support CAS and Action
+  // Cache services.
```
Comment (buchgr) [inline build/bazel/remote/execution/v2/remote_execution.proto]: Does this comment add additional value? If not, could we remove it? Same below.

File: build/bazel/remote/execution/v2/remote_execution.proto (hunk 1 above)
Comment (ola-rozenfeld) [reply build/bazel/remote/execution/v2/remote_execution.proto]: Hmm, I guess this is already an exhaustive list :-)

File: build/bazel/remote/execution/v2/remote_execution.proto (hunk 2 above)
Comment (ola-rozenfeld) [reply build/bazel/remote/execution/v2/remote_execution.proto]: Sure.

**Comment by edbaunton** [comment]
Maybe I missed this in a separate discussion but is it the case that we cannot host the ActionCache separately from the CAS?

**Comment by ola-rozenfeld** [comment]
Correct, these two are linked.

**Pull request merged**